### ISSUE TITLE
[BUG] Clic in the dropzone cause a re-render even with noClick

### ISSUE THREAD

**Issue by Ganbin** [react-dropzone/react-dropzone#1306]
**Describe the bug**
If we use the `noClick` prop, the clic still cause a re-render whenever we clic inside the dropzone

**To Reproduce**
1. Go to [codesandbox example](https://codesandbox.io/s/jovial-bardeen-xttoc7?file=/src/styles.css)
2. Show the console.
3. Click on inside and outside of the dropzone (red box)
5. See the re-render is trigered 

**Expected behavior**
The dropzone should not re-render

**Desktop (please complete the following information):**
 - OS: Mac / Windows
 - Browser Chrome / Safari
 - Version ^12.1.0

**Additional context**
If you keep clicking while inside the zone, it will not always re-render, but clicking inside then outside will always cause the re-render.

It is related to #1035 but I think it was not the same as in my case it is related to the `noClick`.

**Comment by Ganbin** [comment]
I have found that adding 
```
onFocus={null}
onBlur={null}
```
to my div will fix the issue.

I think the `onFocus` and `onBlur` event should not trigger a re-render.

**Comment by vandijkstef** [comment]
I have the same issue. It's a pretty nasty bug as, in my setup, the (mouse-based) interactions done within the dropzones children are negated due to the re-render.

The solution provided by @Ganbin to `null` the onFocus and onBlur prop does work. If fully fixing the bug is an issue; Having these props on the `DropZoneOptions` ánd returned by `getRootProps()` would allow me to integrate it in a wrapping hook. Being unspecified like `passProps: React.HTMLProps<HTMLElement>` would do the trick too.

**Comment by Lexpeee** [comment]
I also have this same problem, and adding the `onFocus` and the `onBlur` prop didn't budge at all. My images still re-render after doing the same scenario. Maybe there's something I've missed. 🤔

**Comment by rolandjitsu** [comment]
> I have found that adding
> 
> ```
> onFocus={null}
> onBlur={null}
> ```
> 
> to my div will fix the issue.
> 
> I think the `onFocus` and `onBlur` event should not trigger a re-render.

That is correct. You'd need to prevent all other events that are captured/have handlers in order to prevent re-renders.

**Comment by rolandjitsu** [comment]
Let me have a quick look at this.
